diff -aurN openvswitch-2.4.0.p/rhel/openvswitch-kmod-rhel6.spec openvswitch-2.4.0/rhel/openvswitch-kmod-rhel6.spec
--- openvswitch-2.4.0.p/rhel/openvswitch-kmod-rhel6.spec        2015-08-22 02:02:38.961245619 +0900
+++ openvswitch-2.4.0/rhel/openvswitch-kmod-rhel6.spec  2015-09-02 14:29:24.461996967 +0900
@@ -21,6 +21,7 @@
 URL:            http://openvswitch.org/
 Source0:        %{oname}-%{version}.tar.gz
 Source1:        %{oname}-kmod.files
+Patch0:         %{oname}-%{version}.patch
 BuildRoot:      %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
 BuildRequires:  %kernel_module_package_buildreqs

@@ -44,6 +45,9 @@
 %prep

 %setup -n %{oname}-%{version}
+%patch0 -p1
+rm configure
+autoreconf
 cat > %{oname}.conf << EOF
 override %{oname} * extra/%{oname}
 override %{oname} * weak-updates/%{oname}
